### PR TITLE
Fixed: exit when media creation failed

### PR DESCRIPTION
**Pull request by ethouris**
Concerns the testing application only.

### CODE REVIEW

File: testing/srt-test-live.cpp [hunk 1]
```diff
@@ -791,6 +791,15 @@ int main( int argc, char** argv )
     alarm(0);
     end_time = time(0);
 
+    if (!src || !tar)
+    {
+        string tarstate = tar ? "CREATED" : "FAILED";
+        string srcstate = src ? "CREATED" : "FAILED";
```
Comment (maxsharabayko) [inline testing/srt-test-live.cpp:797]: `const char*` might be better.

Comment (ethouris) [reply testing/srt-test-live.cpp:797]: This is test app, and I don't think about micro-optimizations in code like that, moreover, it's just to display an error before exiting.

Comment (maxsharabayko) [reply testing/srt-test-live.cpp:797]: Yes, I agree. Still some `const` is better. Even if it is `const string`.

Comment (maxsharabayko) [inline testing/srt-test-live.cpp:797]:
```suggestion
        const string tarstate = tar ? "CREATED" : "FAILED";
        const string srcstate = src ? "CREATED" : "FAILED";
```

Comment (ethouris) [reply testing/srt-test-live.cpp:797]: Oh, for that I agree.

**Pull request merged**